datasets/ru/nsd_isin/crawler.py

```diff
@@ -73,7 +58,11 @@ def crawl_item(context: Context, url: str):
     security.add("sourceUrl", url)
     security.add("topics", "sanction")
     for prop, prop_val in values["security"].items():
-        security.add(prop, prop_val)
+        if prop in ["issueDate", "maturityDate", "createdAt"]:
```

Comment:
Nice, that works and it short and sweet! Let's keep. What would have also worked (just because we talked at lunch):

```
prop_ = security.schema.get(prop)
if prop_ is not None and prop.type.name == 'date':
```